stacktrace-gps.js

```diff
@@ -321,12 +326,13 @@
                         sourceMappingURL = defaultSourceRoot + sourceMappingURL;
                     }
 
-                    return this._getSourceMapConsumer(sourceMappingURL, defaultSourceRoot).then(function(sourceMapConsumer) {
-                        return _extractLocationInfoFromSourceMapSource(stackframe, sourceMapConsumer, sourceCache)
-                            .then(resolve)['catch'](function() {
-                            resolve(stackframe);
+                    return this._getSourceMapConsumer(sourceMappingURL, defaultSourceRoot)
```

Comment:
I included that that in [a separate commit](https://github.com/stacktracejs/stacktrace-gps/pull/53/commits/331754fbd147b4600ecc4da5744ae172e792459f) so that I could tick the "`node_modules/.bin/jscs -c .jscsrc stacktrace-gps.js` passes without errors" box ;)